javalib/src/main/scala/java/time/Duration.scala

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
In my recent Ryu Double.toString PR, @densh asked me to do a cut & paste of the license terms to
LICENSE.txt (and leave copy here in place).  He may ask the same here.

Reply:
I made a nice entry in the license file that I think is even up to your documentation standards 😄

Reply:
Bravo!   You even avoided the word `begat`.  If I were an IP lawyer I could trace that.